Automattic/vip-go-node
# Add Redis helper library

Fixes #110 

Adding a helper library that Node apps can use to instantiate a Redis client compatible with VIP Go.
_Goal:_ Make it easier for clients to use Redis with VIP Go configurations.

### TESTING
1. Run `redis-server`
2. Run `REDIS_MASTER=[HOST]:[PORT] node ./src/redis/index.js`
3. Should log message `Connected to Redis client`
